querydsl-sql/src/test/java/com/querydsl/sql/types/TypeTest.java

```diff
@@ -45,7 +45,29 @@
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-        if (method.getName().startsWith("get")) {
+        if (method.getName().equals("wasNull")) {
```

Comment:
I think he means it more as a [general rule](http://checkstyle.sourceforge.net/config_coding.html#EqualsAvoidNull).
Since there is a checkstyle check, we could incorporate this.